Dockerfile

```diff
@@ -41,6 +41,15 @@ COPY --from=build /build/out/ /
 # Add entrypoint script
 COPY docker-entrypoint.sh /usr/local/bin/
 
+# Create a non-root user, group, directories and switch to that user.
+RUN groupadd -r -g 1001 pushpin && \
+    useradd -r -u 1001 -g pushpin pushpin
+
+RUN mkdir -p /var/run/pushpin && \
+    chown -R pushpin:pushpin /etc/pushpin /var/run/pushpin
```

Comment:
Perhaps `/var/log/pushpin` should also be listed here? It's the default `logdir`.

Reply:
The docker image is configured to output to stdout and not write log files.